pkg/download/automation/automation_test.go

```diff
@@ -28,31 +28,86 @@ import (
 )
 
 func TestDownloader_Download(t *testing.T) {
-	server := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
-		switch req.URL.Path {
-		case "/platform/automation/v1/workflows":
-			wfData, _ := os.ReadFile("./testdata/listWorkflows.json")
-			rw.Write(wfData)
-		case "/platform/automation/v1/business-calendars":
-			wfData, _ := os.ReadFile("./testdata/listBusinessCals.json")
-			rw.Write(wfData)
-		case "/platform/automation/v1/scheduling-rules":
-			wfData, _ := os.ReadFile("./testdata/listSchedulingRules.json")
-			rw.Write(wfData)
-		default:
-			t.Fatal("NO")
-		}
+	t.Run("download all resource", func(t *testing.T) {
+		server := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
+			switch req.URL.Path {
+			case "/platform/automation/v1/workflows":
+				wfData, _ := os.ReadFile("./testdata/listWorkflows.json")
+				rw.Write(wfData)
+			case "/platform/automation/v1/business-calendars":
+				wfData, _ := os.ReadFile("./testdata/listBusinessCals.json")
+				rw.Write(wfData)
+			case "/platform/automation/v1/scheduling-rules":
+				wfData, _ := os.ReadFile("./testdata/listSchedulingRules.json")
+				rw.Write(wfData)
+			default:
+				t.Fatal("NO")
```

Comment:
nit: Not your change, but could we change this to a more useful message please? Just picturing a test failing with "NO" and trying to figure out what exactly when wrong 😅 

e.g.
```suggestion
				t.Fatal("Unexpected API call to %s", req.URL.Path)
```

Reply:
done